client/pom.xml

```diff
@@ -17,7 +17,7 @@
         <slf4j-log4j12.version>1.7.25</slf4j-log4j12.version>
         <slf4j-api.version>1.7.25</slf4j-api.version>
         <junit.version>4.13.1</junit.version>
-        <guava.version>14.0</guava.version>
+        <guava.version>23.0</guava.version>
```

Comment:
Thanks for your contribution. And what's the purpose to update guava?

Reply:
I tried it with guava version 26, but it seems that there is something wrong with nebula-flink.

Reply:
sorry but what the error is? you also need replace getHostText->getHost too.
I just tried nebula-tools exchange and it successed.
Forgive my english not so good.

Reply:
Please indicate whether other users such as exchange、spark-connector、 flink-connector need to make changes.

Reply:
yeah, all it's ok with guava version 23.No modification is required.                                                                                                        but when i upgrade it to 26, there is something else wrong

Reply:
Maybe there is other different with version23 and 26, you can check the error detail and try to soloved it. Or post the error here.